src/stepFunctions/commands/visualizeStateMachine/aslVisualizationManager.ts

```diff
@@ -45,7 +41,7 @@ export class AslVisualizationManager {
         const textDocument: vscode.TextDocument = activeTextEditor.document
 
         // Attempt to retrieve existing visualization if it exists.
-        const existingVisualization = this.getExistingVisualization(textDocument.uri)
+        const existingVisualization = this.getExistingVisualization(textDocument.uri.path)
```

Comment:
General rule: use `fsPath` instead of `path`. This goes for ___all___ instances of `vscode.Uri.path`.